index.js

```diff
@@ -686,6 +686,7 @@ RedisClient.prototype.return_reply = function (reply) {
             throw new Error("subscriptions are active but got an invalid reply: " + reply);
         }
     } else if (this.monitoring) {
+        reply = reply && reply.toString && reply.toString();
         len = reply.indexOf(" ");
```

Comment:
if reply is no t a string type, won't the following lines result in an exception?